sc-app-deploy/sad_deploy/deploy_commands.py

```diff
@@ -0,0 +1,105 @@
+from logging import exception
```

Comment:
It seems that "code copy" is used here instead of improving existing files. The code below is shown as new code to me in the review, but it is existing code. Next time: Improve / rename existing files.

Reply:
This is no copy, but code has been moved from main script sc-app-deploy.py into a new module. This is an improvement as the sequence of calls to deploy image has bben moved in a looop